tests/unit/store/settings.spec.js

```diff
@@ -496,37 +514,38 @@ describe("store/settings", () => {
         user_ids: array_of_userid_1,
       },
     ];
-    expect(store.state.settings.customer_account_ids.length).toEqual(0);
+    expect(store.state.settings.customer_account_ids).toHaveLength(0);
     store.commit("settings/set_customer_account_ids", array_of_customerids);
     store.commit("settings/set_customer_index", 0);
     store.commit("settings/set_user_index", 0);
 
-    let current_customerids = store.getters["settings/customer_account_ids"];
-    let current_list_of_user_ids =
-      store.getters["settings/user_ids_for_selected_customer_account"];
-    expect(current_list_of_user_ids.length).toEqual(1);
+    const current_customerids = store.state.settings.customer_account_ids;
+    const current_list_of_user_ids =
+      store.state.settings.customer_account_ids[0].user_ids;
+    expect(current_list_of_user_ids).toHaveLength(1);
     const add_new_user = {
       user_id: 1,
       uuid: "5FY8KwTsQaUJ2KzHJGetfE",
       nickname: "User account -2",
     };
     /*  (Save ID) user selected */
     current_list_of_user_ids.push(add_new_user);
-    let current_customer_index = store.getters["settings/customer_index"];
+    const current_customer_index = store.state.settings.customer_index;
     current_customerids[
       current_customer_index
     ].user_ids = current_list_of_user_ids;
     /* SaveChanges selected */
     store.commit("settings/set_customer_account_ids", current_customerids);
-    let updated_list_of_user_ids =
-      store.getters["settings/user_ids_for_selected_customer_account"];
-    expect(updated_list_of_user_ids.length).toEqual(2);
+    const updated_list_of_user_ids =
+      store.state.settings.customer_account_ids[current_customer_index]
+        .user_ids;
+    expect(updated_list_of_user_ids).toHaveLength(2);
   });
-  test("Store has a single customer  with multiple user ids can be assigned to the customer_ids array an 'Edit User' ==>  PopopUpDialog (Save ID) ==> SettingsForm (SaveChanges) ", () => {
+  test("Given the Vuex has customer details with multiple user id, When the user updates one of the User ID details, Then validate only modified user ID details are stored in Vuex", () => {
```

Comment:
same comment as previous tests.  the "user" is not doing anything here.  only Vuex mutations are happening.